python/cuml/benchmark/algorithms.py

```diff
@@ -97,7 +101,7 @@ def __init__(
         cpu_data_prep_hook=None,
         cuml_data_prep_hook=None,
         accuracy_function=None,
-        bench_func=fit,
+        bench_func=fit_transform,
```

Comment:
Question: Why change the default to `fit_transform`? What if I want to benchmark only training without inference?

Reply:
The idea was simply for benchmark to cover both training and inference by default. This way, regressions on the inference side won't be missed. This is however, still possible to parameterize an `AlgorithmPair` with an other function to benchmark. Please tell me if you would like to revert that change.

Reply:
I think we want to revert that change and if possible divide training and inference, will make triaging regressions easier and each benchmark to be as simple as possible while still being useful